NEMO/migrations/0013_tool_image_and_description.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2.10 on 2020-02-17 16:59
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('NEMO', '0012_version_2_2_0'),
```

Comment:
that will need to be changed to `0012_version_2_0_0` when you update to latest code. It was a typo that I fixed in the master branch